aws/ecs/packer/to-upload/peers.sh

```diff
@@ -2,29 +2,221 @@
 # This script print a list of IPs of the EC2 instances in your current Autoscaling Group.
 set -eu
 
-aws=/usr/local/bin/aws
+__instance_tag_key="${1:-WEAVE_ECS}"
 
-export AWS_DEFAULT_REGION=$(curl -s curl http://169.254.169.254/latest/dynamic/instance-identity/document | \
-                            jq -r .region)
-current_instance_id=$(curl -s curl http://169.254.169.254/latest/meta-data/instance-id)
-current_autoscaling_group=$($aws autoscaling describe-auto-scaling-instances | \
-                            jq -r ".AutoScalingInstances[] | select(.InstanceId == \"$current_instance_id\") | .AutoScalingGroupName")
+__awscli=
+__instance_id=
+__region=
+__instance_autoscaling_group=
+__peer_autoscaling_instance_ids=
+__list_of_active_spot_fleets=
+__spot_fleet_request_id=
+__spot_fleet_peer_instance_ids=
+__peer_private_ips=
+__peer_tag_instance_ids=
```

Comment:
Please use local variables+echo instead of global variables.

For instance, instead of having a global `region` variable and `get_region` initializing the variable, make `get_region` echo it and initialize `region` with `region=$(get_region ...)`